hosting_docs/source/installation/2-manual-install.rst

```diff
@@ -147,7 +147,7 @@ our example cluster, the control machine is named “control1”.
     ::
 
         $ sudo apt update
-        $ sudo apt install python3-pip python3-venv sshpass net-tools
+        $ sudo apt install python3-dev python3-distutils python3-venv libffi-dev sshpass net-tools
```

Comment:
Did you remove python3-pip because it's installed already by something else? `python3-dev` maybe?

Reply:
Oops. That was actually a mistake. I'll make a new PR.

Reply:
#5904